src/panels/SkyBrowserPanel/WorldWideTelescope/hooks.ts

```diff
@@ -0,0 +1,103 @@
+import { useEffect } from 'react';
+
+import { useAppSelector } from '@/redux/hooks';
+import { useWindowSize } from '@/windowmanagement/Window/hooks';
+
+import {
+  useSelectedBrowserColor,
+  useSelectedBrowserCoords,
+  useSelectedBrowserFov,
+  useSelectedBrowserRadius,
+  useSkyBrowserSelectedImages,
+  useSkyBrowserSelectedOpacities
+} from '../hooks';
+
+import { useWwtProvider } from './WwtProvider/hooks';
+
+// These are the hooks that will keep tabs of the redux state and
```

Comment:
keep tabs?  Do you mean "keep track"?

Since we use the word "tab" for so many things, here especially, I think we should change the phrasing here so it's a bit more clear :)